rulegroup.go

```diff
@@ -171,12 +166,6 @@ RulesLoop:
 		tx.GetCollection(variables.MatchedVars).Reset()
 		tx.GetCollection(variables.MatchedVarsNames).Reset()
 
-		txr := tx.GetCollection(variables.Rule)
```

Comment:
Why did you remove this, is it duplicated? Rules can query their metadata, like

```
SecRule RULE:id "15" "id:15"
```

this should match

Reply:
Yeah, it's duplicated. The code for the same function as follows is used in the subsequent [```r.Evaluate(tx)```](https://github.com/jptosso/coraza-waf/blob/56ff1874903311ba7f497f1fec2c9aaac5ab68e5/rulegroup.go#L180) function. [the same code link](https://github.com/jptosso/coraza-waf/blob/56ff1874903311ba7f497f1fec2c9aaac5ab68e5/rule.go#L249):
```
tx.GetCollection(variables.Rule).SetData(map[string][]string{
	"id":       {strconv.Itoa(rid)},
	"msg":      {r.Msg.String()},
	"rev":      {r.Rev},
	"logdata":  {r.LogData.String()},
	"severity": {r.Severity.String()},
})
```

Reply:
You are right, thank you for noticing